src/amo/components/RatingsByStar/index.js

```diff
@@ -29,11 +30,14 @@ type Props = {|
 
 type InternalProps = {|
   ...Props,
+  clientApp: string,
   dispatch: DispatchFunc,
   errorHandler: ErrorHandlerType,
   groupedRatings?: GroupedRatingsType,
+  history: ReactRouterHistoryType,
   i18n: I18nType,
   location: ReactRouterLocationType,
+  siteLang: ?string,
```

Comment:
I don't think we need the `?` here. `siteLang` should always exist.

Reply:
Ohk! I will change this in the next update.